R/dtc_formats.R

```diff
@@ -0,0 +1,14 @@
+#' Date/time collection formats
+#'
+#' @format A [tibble][tibble::tibble-package] of `r nrow(dtc_formats)` formats
+#'   with three variables:
+#' \describe{
+#'   \item{`fmt`}{Format string.}
+#'   \item{`type`}{Whether a date, time or date-time.}
+#'   \item{`description`}{Description of which date-time components are parsed.}
+#' }
+#'
+#' @examples
+#' dtc_formats
+#'
+"dtc_formats"
```

Comment:
This is a super small dataset. I wonder why you prefer to keep that as generated rd file over just code?

Reply:
Hi @galachad: Indeed the dataset is small right now but likely to grow as we support more formats... I am not sure I understood your point _keep that as generated rd file over just code_? Do you mean keeping the `dtc_formats` as an internal R object only and not export it? The reason for exporting it and to document it is that the users need to find somewhere in the documentation which formats are allowed, and this dataset (and its documentation) provide that.